src/main/java/org/opentestsystem/ap/ims/model/SavedSearchModel.java

```diff
@@ -0,0 +1,89 @@
+package org.opentestsystem.ap.ims.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+
+import java.io.Serializable;
+
+public interface SavedSearchModel {
```

Comment:
These model classes  representing the different data that comes in and goes out of the REST API.